sources/packages/backend/libs/integrations/src/services/sin-validation/sin-validation.service.ts

```diff
@@ -121,9 +121,10 @@ export class SINValidationService extends RecordDataModelService<SINValidation>
           .getOne();
 
         if (!existingValidation) {
-          throw new Error(
-            `Not able to find the SIN validation id ${validationResponse.referenceIndex} to be updated with the ESDC response.`,
-          );
+          return {
+            operationDescription: `Not able to find the SIN validation on line number ${validationResponse.lineNumber} to be updated with the ESDC response.`,
+            record: null,
```

Comment:
The record should be declared optional.
```
export interface SINValidationUpdateResult {
  record?: SINValidation;
  operationDescription: string;
}
```
And this can be returned as
```ts
return {
   operationDescription: `...`,
};
```